api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -580,6 +580,18 @@ private boolean userHasEverLoggedIn(
       // See RW-1488.
       throw new BadRequestException("Changing email is not currently supported");
     }
+    updateInstitutionalAffiliations(updatedProfile, user);
```

Comment:
Do we have a name for the old affiliations? Or did we give the new ones a different name?

Reply:
New ones are Verified Institutional Affiliations

Reply:
👍